### PR TITLE
XIVY-9123 reviewed wording

### PR DESCRIPTION
**Pull request by andreasbalsiger**

### CODE REVIEW

File: src/app/pages/news/9.4/08-tracing-system-overview.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Slow Requests and Traffic Graph {#tracing}
 
 It is key for an orchestration platform like Axon Ivy to provide tools to analyze problems. 
-The new Slow Requests and Traffic Graph views in the Engine Cockpit provide you with all 
+The new "Slow Requests and Traffic Graph" views in the Engine Cockpit provide you with all
```
Comment (weissreto) [inline src/app/pages/news/9.4/08-tracing-system-overview.md:4]: to different views: "Slow Requests" and "Traffic Graph" views

**Review by weissreto — APPROVED** [review]
One small comment. Other looks good.

File: src/app/pages/news/9.4/08-tracing-system-overview.md (hunk 1 above)
Comment (andreasbalsiger) [reply src/app/pages/news/9.4/08-tracing-system-overview.md]: we will go for peters voice

**Review by phochstr — APPROVED** [review]

File: src/app/pages/news/9.4/08-tracing-system-overview.md (hunk 1 above)
Comment (andreasbalsiger) [reply src/app/pages/news/9.4/08-tracing-system-overview.md]: I couldn't remove lli thats why :(

Comment (phochstr) [reply src/app/pages/news/9.4/08-tracing-system-overview.md]: I changed the graphs to italic, and removed TThe Ttypo we had in another file.

**Pull request merged**